content/en/docs/setup/install/helm/index.md

```diff
@@ -0,0 +1,236 @@
+---
+title: Customizable Install with Helm
+description: Install and configure Istio for in-depth evaluation or production use.
+weight: 27
+keywords: [kubernetes,helm]
+owner: istio/wg-environments-maintainers
+icon: helm
+test: no
+---
+
+Follow this guide to install and configure an Istio mesh using
+[Helm](https://helm.sh/docs/) for in-depth evaluation or production use.
+
+The Helm charts used in this guide are the same underlying charts used when
+installing Istio via [Istioctl](/docs/setup/install/istioctl/) or the
+[Operator](/docs/setup/install/operator/).
+
+## Prerequisites
+
+1. [Download the Istio release](/docs/setup/getting-started/#download).
+
+1. Perform any necessary [platform-specific setup](/docs/setup/platform-setup/).
+
+1. Check the [Requirements for Pods and Services](/docs/ops/deployment/requirements/).
+
+1. [Install a Helm client](https://helm.sh/docs/intro/install/) with a version higher than 3.1.
```

Comment:
This is a little ambiguous. Do you mean 3.1.1, or 3.2.0?

Reply:
Added specific version